orchest-sdk/python/orchest/utils.py

```diff
@@ -8,6 +8,22 @@
 from orchest.pipeline import Pipeline
 
 
+def pretty_print_env_var_suggestion(pipeline: Pipeline):
+    print(
+        "Please specify a pipeline step UUID by setting the ORCHEST_STEP_UUID "
+        + "environment variable. Choose one of the following pipeline steps:"
+    )
+    for step in pipeline.steps:
+        print(
+            "ORCHEST_STEP_UUID="
+            + step.properties.get("uuid")
+            + " (Step %s %s)"
+            % (step.properties.get("title"), step.properties.get("file_path"))
+        )
```

Comment:
Nitpick (feel free to choose whatever): in my opinion readability of fstrings is better here.
```suggestion
        print(
            f'ORCHEST_STEP_UUID={step.properties.get("uuid")} '
            f'(Step {step.properties.get("title")}, {step.properties.get("file_path")})'
        )
```

(Wanted to add it to show how fstring usage would look.)

Reply:
Haha still not convinced in this case :D